playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -465,7 +465,7 @@ EDXAPP_JWT_EXPIRATION: 30  # Number of seconds until expiration
 EDXAPP_JWT_AUDIENCE: "lms-key"
 EDXAPP_JWT_SECRET_KEY: "lms-secret"
 
-EDXAPP_PLATFORM_TWITTER_ACCOUNT: ''
+EDXAPP_PLATFORM_TWITTER_ACCOUNT: 'https://lagunita.stanford.edu'
```

Comment:
Should this just be for lagunita (via group_vars)?

Else this could mistakenly sneak onto CME or suclass..

Reply:
we were hardcoding it in platform, so I think we actually just override the whole social networking template for CME and SUclass...

Reply:
This needs a comment on why we're using a URL instead of a twitter handle.

Reply:
like so:
https://github.com/Stanford-Online/openedx-themes/blob/master/cme/lms/templates/courseware/course_about_sidebar_header.html
https://github.com/Stanford-Online/openedx-themes/blob/master/suclass/lms/templates/courseware/course_about_sidebar_header.html

Reply:
There you go, @stvstnfrd